doc/using/configuration.chapter.md

```diff
@@ -164,6 +166,50 @@ There are several ways to tweak how Nix handles a package which has been marked
 
     Note that `permittedInsecurePackages` is only checked if `allowInsecurePredicate` is not specified.
 
+## Installing packages with warnings {#sec-ignore-warnings}
+
+There are several ways to tweak how Nix handles a package which has some warnings.
+
+-   To temporarily allow ignore all warnings, you can use an environment variable for a single invocation of the nix tools:
+
+    ```ShellSession
+    $ export NIXPKGS_IGNORE_WARNINGS=1
+    ```
+
+-   It is possible to permanently but selectively ignore all warnings from specific packages, while still blocking other warnings by default using the `ignoreWarningsPackages` configuration option in the user configuration file.
+
+    The following example configuration permits the installation of the hypothetical package `hello`, version `1.2.3`:
+
+    ```nix
+    {
+      ignoreWarningsPackages = [
+        "hello-1.2.3"
+      ];
+    }
+    ```
+
+-   It is also possible to create a custom policy around which warnings to ignore, by overriding the `ignoreWarningsPredicate` configuration option.
+
+    The `ignoreWarningsPredicate` option is a function which accepts a package and returns a boolean, much like `allowUnfreePredicate`.
+
+    The following configuration example ignores all deprecation warnings:
+
+    ```nix
+    {
+      ignoreWarningsPredicate = pkg: lib.all (issue: issue.kind == "deprecated") (pkg.meta.issues or []);
+    }
+    ```
+
+    Note that `ignoreWarningsPackages` is only checked if `ignoreWarningsPredicate` is not specified.
+
+### `config` Options Reference
+
+The following attributes can be passed in [`config`](#chap-packageconfig).
+
+```{=docbook}
+<include xmlns="http://www.w3.org/2001/XInclude" href="../doc-support/result/config-options.docbook.xml"/>
+```
+
```

Comment:
This duplicates the existing xinclude, causing the reported build failure.
Might have been caused by a bad rebase.

Reply:
Oops, my bad. Thanks for noticing